api_loader.go

```diff
@@ -39,7 +39,17 @@ func prepareStorage() (*RedisClusterStorageManager, *RedisClusterStorageManager,
 	return &redisStore, &redisOrgStore, healthStore, &rpcAuthStore, &rpcOrgStore
 }
 
-func skipSpecBecauseInvalid(spec *APISpec) bool {
+func filterSpec(spec *APISpec) bool {
```

Comment:
I'd prefer `skipSpec`, as otherwise it's hard to tell what `true` is used for without looking at the code